protocol/codec_tester.go

```diff
@@ -57,7 +57,20 @@ func RandomizeObject(template interface{}) (interface{}, error) {
 		return nil, fmt.Errorf("RandomizeObject: must be ptr")
 	}
 	v := reflect.New(tt.Elem())
-	err := randomizeValue(v.Elem(), tt.String(), "")
+	changes := ^int(0)
```

Comment:
are we making up to 2^31 changes by looping back from -1 to -2^31 and then to 0 with overflow?

Reply:
fixed.